### PR TITLE
CASMINST-5220

### PR DESCRIPTION
**Pull request by cdelatte-hpe**
# Description
Fix shebang issue with csm_rbd_tool.py on a csm 1.2 system.  This includes getting the venv on m001 while it is still booted into a 1.2 image.

# Checklist Before Merging

- [x ] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [ ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [ x] My commits or Pull-Request Title contain my JIRA information, or I don't have a JIRA.

[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

File: upgrade/Stage_0_Prerequisites.md [hunk 1]
```diff
@@ -27,23 +27,21 @@ Stage 0 has several critical procedures which prepare the environment and verify
...
 
-    ```bash
-    source /opt/cray/csm/scripts/csm_rbd_tool/bin/activate
-    /usr/share/doc/csm/scripts/csm_rbd_tool/csm_rbd_tool.py --pool_action create --rbd_action create --target_host ncn-m001
-    deactivate
-    ```
+    1. Initialize the Python virtual environment.
 
-    **Note:** This same `rbd` device can be remapped to `ncn-m002` later in the upgrade procedure, when the CSM tarball is needed on that node.
+        Instructions needed
```
Comment (bklei) [inline upgrade/Stage_0_Prerequisites.md:34]: do we want this here?

Comment (mharding-hpe) [reply upgrade/Stage_0_Prerequisites.md:34]: No-- I was going to update that once I got instructions from Craig

Comment (mharding-hpe) [reply upgrade/Stage_0_Prerequisites.md:34]: I think we're going to be remaking this PR. Talking it over with Craig.

Comment (mharding-hpe) [reply upgrade/Stage_0_Prerequisites.md:34]: Yes, new PR forthcoming. Stay tuned.

Comment (mharding-hpe) [reply upgrade/Stage_0_Prerequisites.md:34]: New PR: https://github.com/Cray-HPE/docs-csm/pull/2230

**Pull request closed without merging**